hermetic_build/common/model/gapic_inputs.py

```diff
@@ -19,24 +19,32 @@
 proto_library_pattern = r"""
 proto_library_with_info\(
 (.*?)
-\)
+\\?[\nn]\)
 """
 gapic_pattern = r"""
 java_gapic_library\(
 (.*?)
-\)
+\\?[\nn]\)
 """
 assembly_pattern = r"""
 java_gapic_assembly_gradle_pkg\(
 (.*?)
-\)
+\\?[\nn]\)
 """
 # match a line which the first character is "#".
 comment_pattern = r"^\s*\#+"
-pattern_to_proto = {
-    r"//google/cloud:common_resources_proto": "google/cloud/common_resources.proto",
-    r"//google/cloud/location:location_proto": "google/cloud/location/locations.proto",
-    r"//google/iam/v1:iam_policy_proto": "google/iam/v1/iam_policy.proto",
+# This pattern ignores api definitions that externally reference other repos.
+# For example: @googleapis_repo//google/cloud/...
+pattern_proto_library_to_additional_protos = {
+    r".*//google/cloud:common_resources_proto": "google/cloud/common_resources.proto",
+    r".*//google/cloud/location:location_proto": "google/cloud/location/locations.proto",
+    r".*//google/iam/v1:iam_policy_proto": "google/iam/v1/iam_policy.proto",
+}
+# This is a special case for showcase, where the additional protos
+# are not declared in the gapic library instead
+pattern_gapic_library_to_additional_protos = {
```

Comment:
Do you mind elaborating on why we need this special case?

Reply:
From discussion, let's append the additional protos manually in generation_showcase.sh

Reply:
Actually, we can just append a proto_library_with_info with the protos we need.

Reply:
Done